webapp/components/Firebase/FirebaseInit.ts

```diff
@@ -0,0 +1,53 @@
+// Import the functions you need from the SDKs you need
```

Comment:
remove this comment, there is no need for this

Reply:
Same for other comments from firebase documentation. We don't need the comments in our codebase.